pandas-dev/pandas
# CI: Run sdist build less often

- [ ] closes #43745
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them

Need to test all possible combos of 
- Run on merged commits [tested]
- Run when labeled [tested]
- Skip when no "Build" label [tested]
- Run when commit pushed to labeled PR [tested]
- Run when labeled PR closed/reopened [tested]
